### PR TITLE
ODATA-1636

### PR DESCRIPTION
**Pull request by ralfhandl**

### CODE REVIEW

File: odata-json-format/19 Batch Requests and Responses.md [hunk 1]
```diff
@@ -262,6 +262,48 @@ Content-Length: ###
 ```
 :::
 
+## ##subsec Referencing Response Body Values
+
+::: example
+Example ##ex: a batch request that contains the following operations in
+the order listed:
+the order listed:
```
Comment (HeikoTheissen) [inline odata-json-format/19 Batch Requests and Responses.md:270]:
```suggestion
```

File: odata-json-format/19 Batch Requests and Responses.md [hunk 2]
```diff
@@ -262,6 +262,48 @@ Content-Length: ###
...
+- Get an employee (with `Content-ID = 1`)
+- Get all employees residing in the same building
```
Comment (HeikoTheissen) [inline odata-json-format/19 Batch Requests and Responses.md:272]: Get? The second request is a PATCH request.

File: odata-protocol/11.7 Batch Requests.md [hunk 3]
```diff
@@ -402,6 +402,42 @@ If-Match: $1
...
+
+--batch_36522ad7-fc75-4b56-8c71-56071383e77b
+Content-Type: application/http
+Content-ID: 2
+
+GET /service/Employees?$filter=Building eq $1/Building HTTP/1.1
+Host: host
+Accept: application/json
+
+<Filtered list of employees>
```
Comment (HeikoTheissen) [inline odata-protocol/11.7 Batch Requests.md:436]: A GET request has no body.

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]

File: odata-json-format/19 Batch Requests and Responses.md (hunk 2 above)
Comment (ralfhandl) [reply odata-json-format/19 Batch Requests and Responses.md:272]: Thanks! Forgot to completely "patch" the copied example.

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Pull request merged**